components/scripts/lib/libs.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+readonly BUILD_SCAN_SUMMARY_JAR="${SCRIPT_DIR}/lib/develocity/build-scan-summary-${SUMMARY_VERSION}.jar"
```

Comment:
I put this here because it can only be defined once (since it has a common name that is now used from both build-scan-offline.sh and build-scan-online.sh), and I couldn't think of a better place to put it.

We could create a `build_scan.sh` file, but the only thing it would contain is this constant.

Reply:
In `init.sh` we have a section for "common constants". That seems like a good place?

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/9fadce5a882f99c72a146166e1dffa0f96a3076d/components/scripts/lib/init.sh#L21

Reply:
Good idea! Moved.